src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -244,22 +244,22 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
     private fun checkImpl(holder: AnnotationHolder, impl: RsImplItem) {
         val trait = impl.traitRef?.trait ?: return
         val traitName = trait.name ?: return
-
-        val canImplement = trait.functionList.associateBy { it.name }
-        val mustImplement = canImplement.filterValues { it.isAbstract }
+        
```

Comment:
I mean that we don't need to call `traitRef?.trait` (that resolves trait) anymore since we don't use the trait itself in this method. `toImplementOverride()` will collect all the information we need from the trait and either will return it or will return `null` if it can't resolve the trait. In the latter case we just can return from `checkImpl()` as we do now in `impl.traitRef?.trait ?: return`.